<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>531</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="row0">
       <item>
        <widget class="QLineEdit" name="c00"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c01"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c02"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c03"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c04"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="row0Submit">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row1">
       <item>
        <widget class="QLineEdit" name="c10"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c11"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c12"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c13"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c14"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="row1Submit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row2">
       <item>
        <widget class="QLineEdit" name="c20"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c21"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c22"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c23"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c24"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="row2Submit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row3">
       <item>
        <widget class="QLineEdit" name="c30"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c31"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c32"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c33"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c34"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="row3Submit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row4">
       <item>
        <widget class="QLineEdit" name="c40"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c41"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c42"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c43"/>
       </item>
       <item>
        <widget class="QLineEdit" name="c44"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="row4Submit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="try_next_text">
       <property name="text">
        <string>Try: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="top_results">
       <property name="text">
        <string>Top Results: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>565</width>
     <height>45</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
